### PR TITLE
updated translation system

### PR DESCRIPTION
**Pull request by pecedevelopment**
using TranslateCap function

### CODE REVIEW

**Review by Thekuca — CHANGES_REQUESTED** [review]
client/main.lua: check discord

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+root = true
```
Comment (Gellipapa) [inline .editorconfig:1]: Not add editorconfig please remove this file, and use .gitignore.

File: client/main.lua [hunk 2]
```diff
@@ -48,12 +48,14 @@ function OpenAnimationsSubMenu(menu)
 			title = Config.Animations[i].label
 
 			for j=1, #Config.Animations[i].items, 1 do
+				
 				elements[#elements+1] = {
 					icon = "fas fa-smile",
-					title = Config.Animations[i].items[j].label,
+					title  = Config.Animations[i].items[j].data.index and Config.Animations[i].items[j].label..Config.Animations[i].items[j].data.index or Config.Animations[i].items[j].label,
```
Comment (Gellipapa) [inline client/main.lua:54]: Add variable this is unreadable please.

File: fxmanifest.lua [hunk 3]
```diff
@@ -5,13 +5,16 @@ game 'gta5'
 description 'ESX Animations'
 lua54 'yes'
 
-version '1.0'
+version '1.1'
 legacyversion '1.9.1'
```
Comment (Gellipapa) [inline fxmanifest.lua:9]: Change legacy version please latest.

File: fxmanifest.lua [hunk 4]
```diff
@@ -5,13 +5,16 @@ game 'gta5'
...
 
+
 client_scripts {
 	'@es_extended/imports.lua',
+	'@es_extended/locale.lua',
+	'locales/*.lua',
 	'config.lua',
-	'client/main.lua'
+	'client/main.lua',
```
Comment (Gellipapa) [inline fxmanifest.lua:17]: remove "," please.

**Review by Gellipapa — CHANGES_REQUESTED** [review]
don't remove localization configs revert this changes.

File: .editorconfig (hunk 1 above)
Comment (pecedevelopment) [reply .editorconfig]: done

**Comment by Gellipapa** [comment]
@pecedevelopment Open new pull request i closed this because is so big conflicts.

**Pull request closed without merging**